analyzers/rspec/cs/S4261_c#.html

```diff
@@ -37,4 +37,6 @@ <h2>See</h2>
   <li> <a href="https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap">Task-based
   Asynchronous Pattern</a> </li>
 </ul>
+<h2>Deprecated</h2>
```

Comment:
We should also put something like `[Obsolete("This rule is deprecated in favor of S4790")]` to the class like here https://github.com/SonarSource/sonar-dotnet/blob/1043838c7665a38fd2cb409506f82c5dab4e5a1b/analyzers/src/SonarAnalyzer.CSharp/Rules/InsecureHashAlgorithm.cs#L33-L37